content/en/storage-providers/setup/initialize.md

```diff
@@ -62,7 +62,7 @@ Before you start your miner, it is important to configure it so that it is reach
 ...
 ```
 
-Once you start your miner, [make sure you can connect to its public IP/port]({{< relref "connectivity" >}}).
+Once you start your miner, [make sure you can connect to its public IP/port]({{< relref "../operate/connectivity" >}}).
```

Comment:
I think full paths have already been added to the setup/initialze page, so the changes made here seems to come from an older commit maybe?

Reply:
Yeah. It's part of conflict in this PR. I will keep your paths to resolve it.